data/core/macros/abilities.cfg

```diff
@@ -428,3 +537,49 @@ Enemy units cannot see this unit while it is in deep water, except if they have
         description= _ "This attack poisons living targets. Poisoned units lose 8 HP every turn until they are cured or are reduced to 1 HP. Poison cannot, of itself, kill a unit."
     [/poison]
 #enddef
+
+#define WEAPON_SPECIAL_SHOCK
```

Comment:
This is used in another campaign too, EI IIRC.
With it being moved to core, EI can use this.

Reply:
I have actually removed that EI file already.

Reply:
Great! I didn't see because I wasn't looking at removed files.